java-perf-workshop-server/src/main/java/cchesser/javaperf/workshop/data/Searcher.java

```diff
@@ -94,6 +110,19 @@ public SearchResultElement apply(ConferenceSession input) {
         public List<SearchResultElement> getResults() {
             return results;
         }
+
+        public String getResultsContext() {
+            return resultsContext;
+        }
+
+        @Override
+        public String toString() {
```

Comment:
You don't have to apply this, but I have gotten into a habit to avoid using IDE generated blocks for toString, where we just utilize reflections so it can adapt overtime, and you get the benefit of toString views in logging / troubleshooting.

[ReflectionToStringBuilder](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/builder/ReflectionToStringBuilder.html)

Reply:
It's a good change, I'll adapt it.

Reply:
Any reason why you didn't apply this to your other toString implementations on this PR (ex. ConferenceSession)?

Reply:
Spaced out on what other files I had touched!

Reply:
Done :+1: